cmd/amppkg_dl_sxg/main.go

```diff
@@ -17,6 +19,7 @@ import (
 
 var flagOutSXG = flag.String("out_sxg", "test.sxg", "Path to where the signed-exchange should be saved.")
 var flagOutCert = flag.String("out_cert", "test.cert", "Path to where the cert-chain+cbor should be saved.")
+var flagCertURL = flag.String("cert_url_path", "", "Override scheme, hostname and path in cert-url.")
```

Comment:
Rename to cert_url_base. (Maybe s/path/parent path/ in the flag description?)

Reply:
Fixed.